.github/ci-test-connections.yaml

```diff
@@ -67,6 +67,11 @@ connections:
     conn_type: google_cloud_platform
     description: null
     extra: null
+  - conn_id: gdrive_conn
+    conn_type: google_cloud_platform
+    description: null
+    extra:
+      scopes: "https://www.googleapis.com/auth/drive.readonly"
```

Comment:
Airflow UI had `scope` outside `extras`. We should cross-check this

Reply:
even `scope` outside of `extras` doesn't work.

Reply:
now its working